cmd/link/ocmrole/cmd.go

```diff
@@ -133,9 +133,16 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 	linked, err := r.OCMClient.LinkOrgToRole(orgAccount, roleArn)
 	if err != nil {
 		if errors.GetType(err) == errors.Forbidden || strings.Contains(err.Error(), "ACCT-MGMT-11") {
-			r.Reporter.Errorf("Only organization admin can run this command. "+
+			ocmAccount, localErr := r.OCMClient.GetCurrentAccount()
+			if localErr != nil {
+				r.Reporter.Errorf("Error getting OCM account: %v", localErr)
+				return localErr
```

Comment:
Should we return an error here? or include the account ID in the error message if available.
I guess the API call should work, we rely on it in multiple places.

Reply:
imho, log a warning message is bettern than return an error directly

Reply:
Changed, thanks.